google/cloud/spanner_v1/session.py

```diff
@@ -245,6 +250,7 @@ def execute_sql(
         query_mode=None,
         query_options=None,
         request_options=None,
+        column_info=None,
```

Comment:
Please move `column_info ` in the last otherwise it would be a breaking change if some customer is passing the method arguments in order and not using dictionary to pass arguments.

Reply:
Thanks for pointing this out. Changes done